### PR TITLE
don't show error card in prod & make it fullscreen when running cypress

### LINKED ISSUE #16848: Remove "Something went wrong"
**Describe the bug**
The "Something went wrong" was added to 39.0 to avoid some specific problems, but it's currently showing up too many places and doesn't protect against blank pages.
Some issues have gone from a techdebt or P3 bug to a P2 regression, and it's a bad UX.

It has not helped me at all for debugging purpose, and I still have some issues open, where the dialog is showing, but I have spend several hours on trying to reproduce without success.

**To Reproduce**
https://github.com/metabase/metabase/issues?q=is%3Aissue+%22Something+went+wrong%22+created%3A%3E%3D2021-01-01

**Expected behavior**
Remove it from releases and make it part of dev-builds with a fullscreen layer, which hopefully block any Cypress tests and make sure that we fix the underlying issue quickly.

### PR DESCRIPTION
**Pull request by daltojohnso**
Fixes #16848 

This PR makes it so that the little error card doesn't show up in production.

In dev it is still the same:
Screen Shot 2021-07-06 at 4 14 04 PM

As suggested, I've made it fullscreen so that cypress tests will fail:
Screen Shot 2021-07-06 at 4 14 46 PM

### CODE REVIEW

**Comment by daltojohnso** [comment]
@nemanjaglumac -- do you think making this error card fullscreen when running cypress is fine?

**Comment by nemanjaglumac** [comment]
@daltojohnso absolutely. We want tests to fail as quickly as possible if something goes wrong.

**Review by flamber — APPROVED** [review]
This seems like a good compromise. Hopefully we can make it more helpful in the future, so it can help troubleshooting and report statistics.

File: frontend/src/metabase/env.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const isCypressActive = !!window.Cypress;
```
Comment (nemanjaglumac) [inline frontend/src/metabase/env.js:1]: 🤩

**Pull request merged**